demos/fonts/core-with-promise.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!-- 
+  Instead of async/await, you can use Promise style chains.
+  For example: Vex.Flow.setMusicFont(...).then(onFulfilledCallback) 
+-->
+<html>
+  <body>
+    <div id="output"></div>
+    <!-- The vexflow core build includes no music fonts. -->
+    <script src="../../build/vexflow-core.js"></script>
+    <script>
+      // Vex.Flow.setMusicFont(...fontNames) is an async function that triggers
+      // the loading of the fonts (stored in vexflow-font-xxxx.js).
+      // It returns a Promise that resolves to the loaded music font(s).
+      Vex.Flow.setMusicFont('Petaluma', 'Bravura', 'Gonville', 'Custom').then(onFontsLoaded);
```

Comment:
async / await is the same as promise / then. If you like using `.then(onFulfilledCallback)`, it works fine.

Also notice here that `Flow.setMusicFont(...)` takes a variable number of font family strings. The first argument is the primary music font. The others are fallbacks, in case a glyph is not found in the first music font.